content/post/35-ipfs-companion-2.2.0.md

```diff
@@ -0,0 +1,74 @@
+---
+date: 2018-04-10
+url: 35-ipfs-companion-2-2-0
+title: IPFS Companion 2.2.0 brings window.ipfs to your Browser
+author: lidel
+---
+
+I am pleased to announce to everyone in our community that a new version of our
+browser extension has been released!
+
+![demo of v2.2.0](https://ipfs.io/ipfs/QmdJTmCxwcpoGbEVfT6b9j4RZJWNcF2GQG1Ajf9XB6XtVP)
+
+Why is it worth a blog post you ask? See the overview of key features below.
+
+# 🔦 Highlights
+
+### Your node is exposed as `window.ipfs` on every webpage
+
+Websites can now detect if the property exists in window context and opt-in to
```

Comment:
Might be worth adding a note that `window.ipfs` exposes the same api as js-ipfs-api, and that we'd like to see people building apps that use it, perhaps with a link to one of the demos.

Reply:
It's correct that `window.ipfs` exposes the same API as `js-ipfs-api` so it's fine to say that, and it might be easier for people to understand.

It's _more correct_ to say that it exposes an `interface-ipfs-core` compatible interface, but not everyone will know what that is. Note, `js-ipfs-api` exposes a `interface-ipfs-core` compatible interface too.

I also think it's ok to say it exposes the same API as `js-ipfs-api` since you'll only be using `window.ipfs` from JS anyway.